modules/libcom/src/iocsh/iocsh.cpp

```diff
@@ -927,9 +930,9 @@ static void helpCallFunc(const iocshArgBuf *args)
             fputc('\n', epicsGetStdout());
         iocshTableUnlock ();
 
-        fprintf(epicsGetStdout(),
-                "\n"
-                "Type 'help <command>' to see the arguments of <command>.  eg. 'help db*'\n");
+        fprintf(epicsGetStdout(), "\n"
+            "Type 'help <command>' to see the arguments of <command>, "
+            "eg. 'help db*'\n");
```

Comment:
A minor point, but isn't it preferable for readability for a string constant with two newlines to split across 3 source lines?

Reply:
I was trying to use an 80 column text-wrap width, but given the `ANSI_UNDERLINE()` string below it that's probably too pedantic here. I will fix.

Reply:
The original wording doesn't mention glob patterns at all, and my fix for that grew to a second line anyway:
```patch
         fprintf(epicsGetStdout(), "\n"
-            "Type 'help <command>' to see the arguments of <command>, "
-            "eg. 'help db*'\n");
+            "Type 'help <glob>' for information about commands matching\n"
+            "the name or pattern <glob>, e.g. 'help db*'\n");
```

New commits coming soon.